.wordlist.txt

```diff
@@ -720,3 +720,17 @@ Gan
 Ze
 AppDBMemoryEstimation
 Ivantsiv
+SAIRedis
+Dataflow
+POC
+Azp
+cpu
+dpdk
```

Comment:
we can replace all "dpdk" to "DPDK", so we don't need to update the wordlist here.

Reply:
same for CPU.

Reply:
Done